destination/destination.go

```diff
@@ -0,0 +1,125 @@
+// Copyright © 2022 Meroxa, Inc & Yalantis.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+
+	"github.com/conduitio-labs/conduit-connector-db2/config"
+	"github.com/conduitio-labs/conduit-connector-db2/destination/writer"
+
+	_ "github.com/ibmdb/go_ibm_db" //nolint:revive,nolintlint
+)
+
+// Destination DB2 Connector persists records to a db2 database.
+type Destination struct {
+	sdk.UnimplementedDestination
+
+	writer Writer
+	config config.Config
+}
+
+// New creates new instance of the Destination.
+func New() sdk.Destination {
+	return &Destination{}
+}
+
+// Parameters returns a map of named sdk.Parameters that describe how to configure the Destination.
+func (d *Destination) Parameters() map[string]sdk.Parameter {
+	return map[string]sdk.Parameter{
+		config.KeyConnection: {
+			Description: "Connection string to DB2",
+			Required:    true,
+			Default:     "",
+		},
+		config.KeyTable: {
+			Description: "A name of the table that the connector should write to.",
+			Required:    true,
+			Default:     "",
+		},
+		config.KeyPrimaryKey: {
+			Description: "A column name that used to detect if the target table" +
+				" already contains the record (destination).",
```

Comment:
maybe let the user know that it should be unique too

Reply:
good idea!. I added it